Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -2158,6 +2162,102 @@ function Import-CrmSolution{
     }    
 }
 
+#MergeHoldingSolution
+function Merge-HoldingSolution {
+    [CmdletBinding()]
+	PARAM(
+        [parameter(Mandatory=$true)]
```

Comment:
Can you name this variable $conn to be consistent with the rest?

Reply:
Yep, should maintain consistency.